xtas/tasks/_alpino.py

```diff
@@ -113,7 +113,7 @@ def get_dep(line):
                 tokens=list(tokens.values()))
 
 
-def interpret_token(lemma, word, begin, _end, major_pos, _pos2, pos):
```

Comment:
Alpino produces multiple POS tags per word. I'm not sure if it has any terminology for those, but they need to have some name.